File: 251004/Klimovich/discussion/src/main/java/com/example/discussion/model/Message.java
package com.example.discussion.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.data.cassandra.core.mapping.*;
import org.springframework.data.cassandra.core.cql.PrimaryKeyType;

import java.io.Serializable;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Table("messages")
public class Message {
    @PrimaryKeyColumn(name = "country", type = PrimaryKeyType.PARTITIONED)
    private String country;

    @PrimaryKeyColumn(name = "id", type = PrimaryKeyType.CLUSTERED)
    private long id;

    @Column("content")
    private String content;

    @Column("issue_id")
    private long issueId;

}

File: 251004/Zayats/Distributed_Computing_Labs_4_5/Discussion/Models/Message.cs
using Cassandra.Mapping.Attributes;

namespace Discussion.Models;

[Table("tbl_message")]
public class Message : BaseModel
{
    [ClusteringKey]
    public long Id { get; set; } 
    public long TopicId { get; set; }
    public string Content { get; set; }
}